Assets/Scripts/Game/UserInterfaceWindows/DaggerfallExteriorAutomapWindow.cs

```diff
@@ -1587,6 +1587,7 @@ private void GridButton_OnMouseClick(BaseScreenComponent sender, Vector2 positio
         {
             if (InDragMode())
                 return;
+                DaggerfallUI.Instance.PlayOneShot(SoundClips.ButtonClick);
```

Comment:
Minor remark, I'm not sure what editor you're using, but some indentations are not consistent with the rest of the code, and in a few place like this one can be somewhat confusing (with the since instruction "if" just before).